pkg/shell/base_index.js

```diff
@@ -527,6 +531,16 @@
 
     self.has_oops = false;
 
+    /* dark mode */
+    const style = localStorage.getItem('shell:style') || 'auto';
+    if ((window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches && style === "auto") || style === "dark") {
+        document.documentElement.classList.add('pf-theme-dark');
+        self.dark_mode = true;
+    } else {
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17914-20221201-104014-14c78ccb-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)